### PR TITLE
adding miniTri miniapp

### PR DESCRIPTION
**Pull request by mmwolf**

### CODE REVIEW

File: var/spack/repos/builtin/packages/minitri/package.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+class Minitri(MakefilePackage):
+    """A simple, triangle-based data analytics proxy application."""
+
+    homepage = "https://github.com/Mantevo/miniTri"
+    url      = "https://github.com/Mantevo/miniTri/archive/v1.0.tar.gz"
+
+    version('1.0', '947e296ca408275232f47724267a85ce')
+
+    variant('mpi', default=False, description='Build with MPI support')
```
Comment (davydden) [inline var/spack/repos/builtin/packages/minitri/package.py]: Please make True by default

File: var/spack/repos/builtin/packages/minitri/package.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    version('1.0', '947e296ca408275232f47724267a85ce')
+
+    variant('mpi', default=True, description='Build with MPI support')
+
+    depends_on('mpi', when="+mpi")
+
+    @property
+    def build_targets(self):
+        targets = []
+        if '+mpi' in self.spec:
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/minitri/package.py]: Isn't this backwards?  It's using `mpicc` when `+mpi` is NOT in the spec.

File: var/spack/repos/builtin/packages/minitri/package.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+class Minitri(MakefilePackage):
+    """A simple, triangle-based data analytics proxy application."""
+
+    homepage = "https://github.com/Mantevo/miniTri"
+    url      = "https://github.com/Mantevo/miniTri/archive/v1.0.tar.gz"
+
+    version('1.0', '947e296ca408275232f47724267a85ce')
+
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/minitri/package.py]: We have already a few applications tagged `proxy-app`. Maybe it's worth adding the tag also here?

Comment (tgamblin) [reply var/spack/repos/builtin/packages/minitri/package.py]: that too!

File: var/spack/repos/builtin/packages/minitri/package.py (hunk 2 above)
Comment (junghans) [reply var/spack/repos/builtin/packages/minitri/package.py]: Yes, fixed.

**Pull request merged**